package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@cbinsights/fds",
   "description": "Form: A design system by CB Insights",
   "author": "CB Insights",
-  "version": "0.0.0-development",
+  "version": "11.12.0-development",
```

Comment:
[should] revert this back to `0.0.0-development`. We don't need to bump versions anymore, semantic release will handle that for us now .